.github/main.workflow

```diff
@@ -40,10 +40,9 @@ action "tag images" {
 
 action "push images" {
```

Comment:
> It's recommended to not use the WORKDIR instruction in your Dockerfile. 
https://developer.github.com/actions/creating-github-actions/creating-a-docker-container/
We may want to change this https://github.com/kubernetes-sigs/cluster-api-provider-docker/blob/master/Dockerfile#L16